src/main/java/seedu/address/model/policy/Policy.java

```diff
@@ -75,14 +81,14 @@ public String toString() {
      * @return the string representation of the Policy.
      */
     public String toDisplay(boolean isPersonCard) {
-        if (policyNumber.value.equals(PolicyNumber.DEFAULT_VALUE)) {
+        if (policyNumber.toString().equals(PolicyNumber.DEFAULT_VALUE)) {
             // Policy does not actually exist
             return "No Policy Found";
         }
 
         if (isPersonCard) {
-            return policyNumber.value + "\n" + policyIssueDate.toString() + "\n" + policyExpiryDate.toString();
+            return company + "\n" + policyNumber + "\n" + policyIssueDate + "\n" + policyExpiryDate;
         }
-        return policyNumber.value + ", " + policyIssueDate.toString() + ", " + policyExpiryDate.toString();
+        return company + ", " + policyNumber + ", " + policyIssueDate + ", " + policyExpiryDate;
```

Comment:
Nice! Totally forget about the fact that we can just call the object which by default calls the toString() method